src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -0,0 +1,86 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/ha-card.js";
+import "../../../components/ha-markdown.js";
+
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+interface Config extends LovelaceConfig {
+  content: string;
+  title?: string;
+}
+
+export class HuiMarkdownCard extends HassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  protected config?: Config;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  public getCardSize() {
+    return this.config!.content.split("\n").length;
+  }
+
+  public setConfig(config: Config) {
+    if (!config.content) {
+      throw new Error("Invalid Configuration: Content Required");
+    }
+
+    this.config = config;
+  }
+
+  protected render() {
+    if (!this.config) {
+      return html``;
+    }
+
+    return html`
+      ${this.renderStyle()}
+      <ha-card .header="${this.config.title}">
+        <ha-markdown .content="${this.config.content}"></ha-markdown>
+      </ha-card>
+    `;
+  }
+
+  private renderStyle() {
+    return html`
+      <style>
+        :host {
+          @apply --paper-font-body1;
+        }
+        ha-markdown {
+          display: block;
+          padding: 0 16px 16px;
+          padding-top: ${this.config!.title ? '0' : '16px'};
```

Comment:
We shouldn't make stylesheets dynamic, as it means the whole CSS stylesheet has to be re-rendered. I suggest you use a `styleMap` ([example](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/lovelace/cards/hui-entity-button-card.ts#L80))

Reply:
Note that in that case, you could also use a `classMap` ([example](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/panels/lovelace/cards/hui-glance-card.ts#L89)), which I think I prefer, as that means we keep all styling inside `renderStyle`, meaning it can be overridden easily.

Reply:
Great! Thanks! I'll put in a PR for this as well:

https://github.com/home-assistant/home-assistant-polymer/pull/1801/files#diff-67edbcddc3ac9c8fb4e71c404eb16b6fR60